setup.py

```diff
@@ -1,6 +1,7 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 from setuptools import setup, find_packages
 
+
```

Comment:
Can I ask if this was intentional?

Reply:
I thought it would be required by black/flake8. It doesn't seem to care. I'll revert this.